app/src/pages/project/SpansTable.tsx

```diff
@@ -299,7 +299,7 @@ export function SpansTable(props: SpansTableProps) {
         const span = row.original;
         const { traceId } = span.trace;
         return (
-          <Link to={`traces/${traceId}?selectedSpanNodeId=${span.id}`}>
+          <Link to={`${traceId}?selectedSpanNodeId=${span.id}`}>
```

Comment:
this might be a slightly weird route. E.g. `spans/:traceId`. Though again I guess I don't have a great solution here.